src/main/java/seedu/address/logic/commands/AliasCommand.java

```diff
@@ -0,0 +1,67 @@
+package seedu.address.logic.commands;
+
+import seedu.address.model.Aliases;
+import seedu.address.model.UserPrefs;
+
+//@@author fishTT
+/**
+ * Creates an alias for other commands.
+ */
+public class AliasCommand extends UndoableCommand {
```

Comment:
Remove this file.
Note that if you wish to claim credit you can move it to an `unused` folder and update the collate tag instead.

![image](https://user-images.githubusercontent.com/875036/38698242-02ccbe46-3ec7-11e8-9f76-64acb53d304f.png)